apps/Tabby/description

```diff
@@ -0,0 +1,4 @@
+ A terminal for a more modern age 
```

Comment:
Try to be more descriptive and explain why it's better than alacritty, eDEX-UI, Hyper, lxterminal, guake, etc.